parts/windows/kuberneteswindowssetup.ps1

```diff
@@ -296,7 +301,11 @@ try
     Get-ProvisioningScripts
     Get-LogCollectionScripts
     
-    Write-KubeClusterConfig -MasterIP $MasterIP -KubeDnsServiceIp $KubeDnsServiceIp
+    Write-KubeClusterConfig `
+        -MasterIP $MasterIP `
+        -KubeDnsServiceIp $KubeDnsServiceIp `
+        -EnableSecureTLSBootstrapping $global:EnableSecureTLSBootstrapping `
+        -CustomSecureTLSBootstrapAADResource $global:CustomSecureTLSBootstrapAADResource

```

Comment:
we only can change it after changing the function in cse script package and updating the cse script package version.

Reply:
made a note about this in the description :)

Reply:
It needs to be separated into two PRs.
 1. update the scripts in staging/cse/windows; 
 2. release a new cse script package
 3. update kuberneteswindowssetup.ps1 with the new cse script package version